doc/kernel.rst.kerneldoc

```diff
@@ -0,0 +1,33 @@
+..
+    Zephyr Project Kernel Documentation master file
+
+Zephyr Project Kernel Documentation
+###################################
+
+This document provides a general introduction of the Zephyr kernel's
+key capabilities and services.
+
+.. toctree::
+   :maxdepth: 3
+
+   kernel/kernel.rst
+
+.. toctree::
+   :maxdepth: 2
+
+   api/api.rst
+
+.. toctree::
+   :maxdepth: 1
+
+   reference/kconfig/index.rst
+
+Links
+^^^^^
+
+.. _application:
+
+Application Development Primer
+""""""""""""""""""""""""""""""
+
+https://docs.zephyrproject.org/latest/application/application.html
```

Comment:
This could use a sentence around the document link rather than just a bare link.